spyder/plugins/debugger/widgets/main_widget.py

```diff
@@ -292,9 +273,7 @@ def setup(self):
         # Options menu
         options_menu = self.get_options_menu()
         for item in [
```

Comment:
```suggestion
        for item in [exclude_internal_action]:
```

Leave this in a single line.